app/client/pages/CreateListingPage.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Card from 'react-bootstrap/Card';
+import logo from '../../assets/images/logo/black_transparent.svg';
+import CreateListing from '../components/CreateListing.jsx';
+import Redirect from '../components/routing/Redirect.jsx';
+import Splash from '../components/layouts/Splash.jsx';
+import { withSession } from '../components/Remote.jsx';
+import selectors from './CreateListingPage.module.scss';
+
+/** Login */
+class CreateListingPage extends React.Component {
+  /** @param {Object} props */
+  constructor(props) {
+    super(props);
+    this.handleAddListing = this.handleAddListing.bind(this);
+  }
+
+  /** @param {Object} credentials */
+  handleAddListing(credentials) {
+    this.props.postRemote('/sessions', {
+      body: credentials,
+      success: (response) => this.props.updateSession(response.body || {})
+    });
+  }
+
+  /** @return {JSX.Element} */
+  render() {
+    return (
+      <>
+        {
+          (this.props.session?.user) ?
+            (<Redirect back/>) :
+            (
+              <Splash id={selectors['addListing']}>
+                <Card id={selectors['Card']} border="primary" style={{ width: '30rem'}}>
+                  <Card.Header>
+                    <Card.Img src={logo}/> Create Listing
```

Comment:
Personal opinion - I don't think the logo needs to be here for this form
```suggestion
                    Create Listing
```